api_app/views.py

```diff
@@ -416,11 +416,17 @@ def pivot(self, request, pk=None, pivot_config_pk=None):
     )
     @cache_action_response(timeout=60 * 5)
     def aggregate_status(self, request):
+        org_param = request.GET.get("org", "").lower() == "true"
+        users_of_organization = None
+        if org_param:
+            users_of_organization = self.get_org_members(request.user)
```

Comment:
this could be a method returning `users_of_organization`, right?

Reply:
yes returns all the members in the organisation

Reply:
I meant that those 4 lines could be a method which returns `users_of_organization`.

I am telling you this because I saw the code repeated multiple times

Reply:
@shivam-Purohit onceyou get some time, if you could just do this little refactor then we can merge this PR :) thank you

Reply:
Sorry i missed the comment above earlier. I think the new refactor should be good.